### ISSUE TITLE
<kbd>Ctrl</kbd> in GFM mode looks different from GitHub.com

### ISSUE THREAD

**Issue by vejuhust** [revolunet/sublimetext-markdown-preview#271]
For such HTML tags in [this markdown file](https://raw.githubusercontent.com/vejuhust/config-box/master/sublime-text/README.md), `<kbd>` tags in local preview page looks different from its counterpart on GitHub.com.
The local one looks more gradient, whereas the GitHub one looks flat. Like this: <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>P</kbd>

Here's my user preferences of Markdown Preview:
```
{
	"build_action": "browser",
	"enable_autoreload": false,
	"github_mode": "gfm",
	"markdown_filetypes":
	[
		".markdown",
		".md",
		".mdown"
	],
	"parser": "github",
	"trim_trailing_white_space_on_save": false,
}
```

**Comment by facelessuser** [comment]
Github can't stop touching their style.  They seem to always be changing something.  I think you are always going to be lagging behind Github's latest style.

The Github CSS we use is ripped from here https://github.com/sindresorhus/github-markdown-css.  He has to keep updating it every couple of months because Github can't leave things alone.  I'm not sure how often the author of Markdown Preview plugin feels about constantly chasing the latest.  You are welcome to try a pull request as long as you make sure you don't break anything.  What we do is pull the latest CSS from the above mentioned repo.  We may add a couple of thinks, but maybe not; I can't remember.  You can diff the two to see the kinds of differences.

**Comment by revolunet** [comment]
thanks both of you, fixed in PR #273

**Comment by taohex** [comment]
<kbd>Ctrl</kbd>
